### PR TITLE
Annotate System.IO.Packaging for nullable reference types

### PR DESCRIPTION
**Pull request by stephentoub**
Contributes to https://github.com/dotnet/runtime/issues/2339
cc: @buyaa-n, @carlossanlop, @ericstj, @twsouthwick

### CODE REVIEW

**Review by buyaa-n — APPROVED** [review]
Left one NIT, overall looks great, thanks!

**Review by carlossanlop — APPROVED** [review]

File: src/libraries/System.IO.Packaging/ref/System.IO.Packaging.cs [hunk 1]
```diff
@@ -73,13 +73,13 @@ public abstract partial class PackagePart
     {
         protected PackagePart(System.IO.Packaging.Package package, System.Uri partUri) { }
         protected PackagePart(System.IO.Packaging.Package package, System.Uri partUri, string contentType) { }
```
[the remark this answers is not in the document]
Comment (stephentoub) [reply src/libraries/System.IO.Packaging/ref/System.IO.Packaging.cs:75]: Thanks, will change it.

**Comment by twsouthwick** [comment]
@stephentoub Should this target .NET Standard 2.1 as well so that nullable annotations will be available?

**Comment by stephentoub** [comment]
@twsouthwick, that shouldn't be necessary, e.g. a .NET 5 app pulling in the System.IO.Packaging package containing these changes should see everything NRT-enabled.  Are you seeing otherwise?

**Comment by twsouthwick** [comment]
I'm looking at adding support for NRT in DocumentFormat.OpenXml and currently support .NET Standard and was hoping to support NRT by adding .NET Standard 2.1 for .NET Core 3.1/blazor users.

**Comment by stephentoub** [comment]
It should "just work" even without a netstandard2.1 asset.

**Comment by twsouthwick** [comment]
Cool I'll give that a try

**Pull request merged**